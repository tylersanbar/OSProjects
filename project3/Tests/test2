#!/bin/bash
# MKD test (simple)
./myfs_format 64
./myfs_inspect -volume
./myfs_inspect -iblock 1
./myfs_inspect -dir 2
./myfs_mkd home
./myfs_inspect -volume
./myfs_inspect -iblock 1
./myfs_inspect -dir 2
./myfs_inspect -dir 3
./myfs_mkd home/bob
./myfs_inspect -volume
./myfs_inspect -iblock 1
./myfs_inspect -dir 2
./myfs_inspect -dir 3
./myfs_inspect -dir 4
./myfs_mkd home/bob/mail
./myfs_inspect -volume
./myfs_inspect -iblock 1
./myfs_inspect -dir 2
./myfs_inspect -dir 3
./myfs_inspect -dir 4
./myfs_inspect -dir 5




