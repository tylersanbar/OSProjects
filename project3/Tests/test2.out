N_BLOCKS: 64
N_ALLOCATED_BLOCKS: 3
N_ALLOCATED_INDEX_NODES: 1
Block allocation table:
00: 07
01: 00
02: 00
03: 00
04: 00
05: 00
06: 00
07: 00
Relative Index Node 0	DIRECTORY	Nref=1	Content=2	size=2
Next block: 65534
Directory at block 2:
Entry 0: name=".", index_node=0
Entry 1: name="..", index_node=0
Next block: 65534
N_BLOCKS: 64
N_ALLOCATED_BLOCKS: 4
N_ALLOCATED_INDEX_NODES: 2
Block allocation table:
00: 0f
01: 00
02: 00
03: 00
04: 00
05: 00
06: 00
07: 00
Relative Index Node 0	DIRECTORY	Nref=1	Content=2	size=3
Relative Index Node 1	DIRECTORY	Nref=1	Content=3	size=2
Next block: 65534
Directory at block 2:
Entry 0: name=".", index_node=0
Entry 1: name="..", index_node=0
Entry 2: name="home", index_node=1
Next block: 65534
Directory at block 3:
Entry 0: name=".", index_node=1
Entry 1: name="..", index_node=0
Next block: 65534
N_BLOCKS: 64
N_ALLOCATED_BLOCKS: 5
N_ALLOCATED_INDEX_NODES: 3
Block allocation table:
00: 1f
01: 00
02: 00
03: 00
04: 00
05: 00
06: 00
07: 00
Relative Index Node 0	DIRECTORY	Nref=1	Content=2	size=3
Relative Index Node 1	DIRECTORY	Nref=1	Content=3	size=3
Relative Index Node 2	DIRECTORY	Nref=1	Content=4	size=2
Next block: 65534
Directory at block 2:
Entry 0: name=".", index_node=0
Entry 1: name="..", index_node=0
Entry 2: name="home", index_node=1
Next block: 65534
Directory at block 3:
Entry 0: name=".", index_node=1
Entry 1: name="..", index_node=0
Entry 2: name="bob", index_node=2
Next block: 65534
Directory at block 4:
Entry 0: name=".", index_node=2
Entry 1: name="..", index_node=1
Next block: 65534
N_BLOCKS: 64
N_ALLOCATED_BLOCKS: 6
N_ALLOCATED_INDEX_NODES: 4
Block allocation table:
00: 3f
01: 00
02: 00
03: 00
04: 00
05: 00
06: 00
07: 00
Relative Index Node 0	DIRECTORY	Nref=1	Content=2	size=3
Relative Index Node 1	DIRECTORY	Nref=1	Content=3	size=3
Relative Index Node 2	DIRECTORY	Nref=1	Content=4	size=3
Relative Index Node 3	DIRECTORY	Nref=1	Content=5	size=2
Next block: 65534
Directory at block 2:
Entry 0: name=".", index_node=0
Entry 1: name="..", index_node=0
Entry 2: name="home", index_node=1
Next block: 65534
Directory at block 3:
Entry 0: name=".", index_node=1
Entry 1: name="..", index_node=0
Entry 2: name="bob", index_node=2
Next block: 65534
Directory at block 4:
Entry 0: name=".", index_node=2
Entry 1: name="..", index_node=1
Entry 2: name="mail", index_node=3
Next block: 65534
Directory at block 5:
Entry 0: name=".", index_node=3
Entry 1: name="..", index_node=2
Next block: 65534
